app/controllers/questions_controller.rb

```diff
@@ -38,21 +38,25 @@ def edit
   def create
```

Comment:
Are you sure that SP is working properly with cancancan? CCC inits the object for you in a before filter. Is it using the sanitized params?